packages/web/src/components/stake/select-lilquidity-list-item/SelectLiquidityListItem.tsx

```diff
@@ -17,26 +18,29 @@ interface SelectLiquidityListItemProps {
   onCheckedItem: (checked: boolean, path: string) => void;
 }
 
+const TokenValueWrapper = styled.div``;
+const TokenTitleWrapper = styled.div``;
```

Comment:
Can you move it to styles.ts in the same folder?
And please put the flex style in there, so we can be predictable.